docs/analytics/auto_scroll_tracker.md

```diff
@@ -20,14 +20,34 @@ AutoScrollTracker is a GOVUK.Module and can be initialised by adding the relevan
 
 By default, AutoScrollTracker tracks by percentage scrolled. Specifically, it will fire a GA event when the user scrolls to 20%, 40%, 60%, 80% and 100% of the page height.
 
-It can track headings instead of percentages, using this additional option:
+It can track headings instead of percentages, using the `data-track-type` option.
 
 ```html
 <% content_for :extra_head_content do %>
   <meta name="govuk:scroll-tracker" content="" data-module="auto-scroll-tracker" data-track-type="headings"/>
 <% end %>
 ```
 
+If only some headings need to be tracked, they can be specified using the `data-track-headings` option as shown. This option has been included with the module to support legacy tracking. It is not recommended for future use as headings on pages can change, which would prevent the tracking from working.
+
+```html
+<% content_for :extra_head_content do %>
+  <meta name="govuk:scroll-tracker" content="" data-module="auto-scroll-tracker" data-track-type="headings" data-track-headings="['Example heading', 'Another example heading']"/>
```

Comment:
I imagine the example here might be better illustrated by using Ruby to_json. 
```suggestion
  <meta name="govuk:scroll-tracker" content="" data-module="auto-scroll-tracker" data-track-type="headings" data-track-headings='<%= ["Example heading", "Another example heading"].to_json %>' />
```

as it seems a bit dubious suggesting people handroll their JSON. Better still as:

```
<%= tag.meta name: "govuk-scroll-tracker", content: "", data: { module: "auto-scroll-tracker", "track-type": "headings", "track-headings": ["Example heading", "Another example heading"].to_json } %>
```

Reply:
Great, thanks, have updated.

Reply:
Also have removed the JSON quote fix - I was trying to make the script more flexible, but if you don't think it's necessary that's fine.

Reply:
Super - I hope it'll be ok. It's potentially a risk, but if the script is forgiving then it creates an impression of JSON flexibility which the language doesn't have which may catch people out any other places.